btsieve/src/bitcoin/bitcoind_http_blocksource.rs

```diff
@@ -0,0 +1,205 @@
+use crate::blocksource::{self, BlockSource};
```

Comment:
Instead of `btsieve/src/bitcoin/bitcoind_http_blocksource.rs` shall we have a `btsieve/src/bitcoin/blocksource/` folder that contains all implementations such as `bitcoind_http.rs` ?